components/profile/ProfileEmptyResult.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <section class="section has-text-centered">
+    <div class="title is-4">
+      {{ $t('profile.searchNoResultsTitle') }}
+    </div>
+    <div v-if="totalChainsLength" class="subtitle is-6">
+      {{ $t('profile.searchNoResultsText', [totalChainsLength]) }}
+    </div>
+    <NeoButton
+      v-if="totalChainsLength"
+      variant="pill"
+      :label="$t('profile.searchNoResultsButton', [chainNames[nextPrefix]])"
+      @click="switchToPrefix(nextPrefix)" />
+  </section>
+</template>
+
+<script lang="ts" setup>
+import { NeoButton } from '@kodadot1/brick'
+import { type Prefix, chainNames } from '@kodadot1/static'
+
+const props = defineProps({
+  hasAssetPrefixList: { type: Array<Prefix>, default: () => [] },
+})
+
+const { redirectAfterChainChange } = useChainRedirect()
+const { setUrlPrefix } = usePrefix()
+const totalChainsLength = computed(() => props.hasAssetPrefixList.length)
+const nextPrefix = computed(() => {
+  return props.hasAssetPrefixList[0]
+})
```

Comment:
what if empty?
```suggestion
const nextPrefix = computed(() =>  props.hasAssetPrefixList[0])
```

Reply:
remove blank
```suggestion
const nextPrefix = computed(() => props.hasAssetPrefixList[0])
```

Reply:
> what if empty?

If it's empty, this component will not be rendered. so, it would not happen : )